contents/handbook/people/hiring-process/index.md

```diff
@@ -272,3 +272,11 @@ If a candidate needs visa sponsorship, please keep in mind that the process is l
 ## E-Verify
 
 We participate in [E-verify](https://www.e-verify.gov/about-e-verify) for all US new hires which allows us to verify employment eligibility remotely and continue hiring in multiple states. E-Verify is not used as a tool to pre-screen candidates.
+
+## Location - April 2024 update
+
+For some teams, it's important to have a wide range of timezones covered by the small team - this allows us to have closer to 24 hour coverage in case of incidents, and is particular relevant ie for infrastructure or pipeline teams.
```

Comment:
```suggestion
For some teams, it's important to have a wide range of timezones covered by the small team - this allows us to have closer to 24 hour coverage in case of incidents, and is particularly relevant for infrastructure or pipeline teams.
```